<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamForm</class>
 <widget class="QDialog" name="ExamForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(139,233,159);
color: rgb(85, 13, 57);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="logButton">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Я вже зареєстрований</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="nickLabel">
               <property name="text">
                <string>Нікнейм:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="nickEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
		background-color: rgb(186, 189, 182);
}
 QLineEdit:enabled{
	
	background-color: rgb(238, 238, 236);
}

*[valid=&quot;true&quot;]{
background-color: white;
border: 2px solid;
border-color: green;
}

*[valid=&quot;false&quot;]{
background-color: white;
border: 2px solid;
border-color: red;
}</string>
               </property>
               <property name="placeholderText">
                <string>Введіть ваш нікнейм</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="passwordlabel">
               <property name="text">
                <string>Пароль:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="passwordEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
		background-color: rgb(186, 189, 182);
}
 QLineEdit:enabled{
	
	background-color: rgb(238, 238, 236);
}

*[valid=&quot;true&quot;]{
background-color: white;
border: 2px solid;
border-color: green;
}

*[valid=&quot;false&quot;]{
background-color: white;
border: 2px solid;
border-color: red;
}</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Введіть ваш пароль</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="regButton">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Я складаю іспит вперше</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="nameLabel">
               <property name="text">
                <string>Ім'я:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="nameEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
		background-color: rgb(186, 189, 182);
}
 QLineEdit:enabled{
	
	background-color: rgb(238, 238, 236);
}

*[valid=&quot;true&quot;]{
background-color: white;
border: 2px solid;
border-color: green;
}

*[valid=&quot;false&quot;]{
background-color: white;
border: 2px solid;
border-color: red;
}</string>
               </property>
               <property name="placeholderText">
                <string>Введіть ваше ім'я</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="surnameLabel">
               <property name="text">
                <string>Прізвище:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="surnameEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
		background-color: rgb(186, 189, 182);
}
 QLineEdit:enabled{
	
	background-color: rgb(238, 238, 236);
}

*[valid=&quot;true&quot;]{
background-color: white;
border: 2px solid;
border-color: green;
}

*[valid=&quot;false&quot;]{
background-color: white;
border: 2px solid;
border-color: red;
}</string>
               </property>
               <property name="placeholderText">
                <string>Введіть ваше прізвище</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="emailLabel">
               <property name="text">
                <string>Email :</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="emailEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
		background-color: rgb(186, 189, 182);
}
 QLineEdit:enabled{
	
	background-color: rgb(238, 238, 236);
}

*[valid=&quot;true&quot;]{
background-color: white;
border: 2px solid;
border-color: green;
}

*[valid=&quot;false&quot;]{
background-color: white;
border: 2px solid;
border-color: red;
}</string>
               </property>
               <property name="placeholderText">
                <string>Введіть ваш  email</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="nickLabel_2">
               <property name="text">
                <string>Нікнейм:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="nickEdit_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
		background-color: rgb(186, 189, 182);
}
 QLineEdit:enabled{
	
	background-color: rgb(238, 238, 236);
}

*[valid=&quot;true&quot;]{
background-color: white;
border: 2px solid;
border-color: green;
}

*[valid=&quot;false&quot;]{
background-color: white;
border: 2px solid;
border-color: red;
}</string>
               </property>
               <property name="placeholderText">
                <string>Введіть ваш нікнейм</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="passwordlabel_2">
               <property name="text">
                <string>Пароль:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="passwordEdit_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit:disabled{
		background-color: rgb(186, 189, 182);
}
 QLineEdit:enabled{
	
	background-color: rgb(238, 238, 236);
}

*[valid=&quot;true&quot;]{
background-color: white;
border: 2px solid;
border-color: green;
}

*[valid=&quot;false&quot;]{
background-color: white;
border: 2px solid;
border-color: red;
}</string>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Введіть ваш пароль</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: rgb(204, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backButton">
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
border-width: 0px;
border-radius: 20%;
}
QPushButton {
padding: 30px;
border: 4px solid;
	border-color: rgb(52, 101, 164);
border-radius: 6px;
	background-color: rgb(189, 214, 254);
	color: rgb(9, 4, 32);
padding: 10px;
outline: none;
}

QPushButton:flat {
border: none; 
}

QPushButton:default {
border-color: navy; 
}</string>
           </property>
           <property name="text">
            <string>До головного меню</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="emptyLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="examButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	border-width: 0px;
	border-radius: 20%;
}
QPushButton {
	border: 4px solid;
	border-color: rgb(52, 101, 164);
	border-radius: 6px;
	background-color: rgb(189, 214, 254);
	color: rgb(9, 4, 32);
	padding: 10px;
	outline: none;
}

QPushButton:flat {
	border: none; 
}

QPushButton:default {
border-color: rgb(92, 53, 102);
}

QPushButton:disabled{
background-color: rgb(211, 215, 207);
}</string>
           </property>
           <property name="text">
            <string>Розпочати іспит</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>ExamForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
